pocket_coffea/lib/triggers.py

```diff
@@ -55,7 +55,8 @@ def get_trigger_mask(events, trigger_dict, year, isMC, primaryDatasets=None, inv
             )
             trigger_mask = trigger_mask | (events.HLT[trigger] & flag)
         else:
-            trigger_mask = trigger_mask | events.HLT[trigger.lstrip("HLT_")]
+            if trigger in events.HLT.fields:
```

Comment:
I think that here we are missing an expection: in case the trigger is not present in the fields,  this would fail  __silently__.  Instead without this if, coffea would emit an exception about a missing filter and the processing would stop. 

I think we can avoid the if and relying on the clear expection in the case of missing trigger.

Reply:
Ah ok I see the problem. But triggers are defined by data taking period. 
Are those triggers present only on part of the NanoAOD files of a given campaign?   As fare as I remember the branches are uniform in the NanoAOD file for a given data/MC campaign. 

We can discuss more about this in a dedicated issue I agree.